.gitignore

```diff
@@ -337,4 +337,31 @@ ASALocalRun/
 .localhistory/
 
 # BeatPulse healthcheck temp database
-healthchecksdb
\ No newline at end of file
+healthchecksdb
+/Practice/Practice/Views/_ViewStart.cshtml
+/Practice/Practice/Views/Shared/_Layout.cshtml
+/Practice/Practice/Views/Shared
+/Practice/Practice/Views/Employee/Create.cshtml
```

Comment:
czy aby na pewno zmiany w tych plikach mają być niewidoczne dla gita?

Reply:
Tak to było tylko dla testu.